src/Messages/InvalidKeyMessages/TSInvalidIdentityKeyErrorMessage.m

```diff
@@ -29,6 +25,11 @@ - (NSString *)theirSignalId
     return nil;
 }
 
+- (BOOL)isDynamicInteraction
+{
+    return YES;
```

Comment:
I was wrestling with whether or not to do this.  My thoughts were:

* It's possible these will become dynamic - we might delete them to de-duplicate them, we might change them to reflect whether or not they have been verified - but it's true they are not dynamic yet.
* These are a lot like the dynamic interactions - rare but of great interest.
* Database views and queries aren't free and I foresaw us often querying for the dynamic interactions and safety number changes at the same time and in the same way. 

So it was a bit of "let's be pragmatic and not puritanical"... but you've talked me out of it.  I've added another database view.